helm/e2e-app-chart/templates/deployment.yaml

```diff
@@ -15,7 +15,16 @@ spec:
       labels:
         app: e2e-app
     spec:
-      serviceAccountName: e2e-app
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchExpressions:
+                  - key: "app"
+                    operator: In
+                    values:
+                    - e2e-app
+              topologyKey: "kubernetes.io/hostname"
```

Comment:
Adding anti affinity. We want to run a "HA" setup to ensure requests are not dropped even if nodes scale/fail.